tests/ui/attributes/mixed_export_name_and_no_mangle.rs

```diff
@@ -0,0 +1,9 @@
+#![deny(mixed_export_name_and_no_mangle)]
```

Comment:
You might also need to do a `build-fail` instead of default `check-fail` since the check  happens very late, i.e. add `//@ build-fail`.

Reply:
as it turned out, `check-fail` works but `build-fail` did not work. So I kept the test as it was.